Patches/GDAL/2.3.2/Patch.cmake

```diff
@@ -0,0 +1,12 @@
+#+
+# This file is called as CMake -P script for the patch step of
+# External_GDAL.cmake.
+# GDAL_patch and GDAL_source are defined on the command line along with the
+# call.
+#-
+
+# Fix the version number to not contain "dev" and add a SOVERSION
+# for the public C API
+message("Copying ${GDAL_patch}/frmts/mrsid/nmake.opts to ${GDAL_source}/frmts/mrsid")
```

Comment:
This message seems wrong ... copy/paste from a different patch file?

Reply:
Everything else looks good. I will run this through the dashboards after this fix. Assuming it passes, I will merge after.

Reply:
Yes, indeed. 
Removed.

Reply:
The comment was a good catch too, but I was speaking about the CMake message

message("Copying ${GDAL_patch}/frmts/mrsid/nmake.opts to ${GDAL_source}/frmts/mrsid")

That isn't correct either.

Reply:
Sorry.

Reply:
No worries, thanks.